### ISSUE TITLE
Polymer2 support 

### ISSUE THREAD

**Issue by florent1933** [SimpleElements/simple-modal#3]
Now Polymer2 is released, have you planned to release your webcomponents in a polymer2 version?

**Comment by seaneking** [comment]
Definitely. These will all be ported when [Simpla](https://www.simpla.io)'s elements are, but would also happily accept PRs in the meantime. Most should be pretty trivial to make into hybrids.

The exception is simple-button, which is blocked by [this PR](https://github.com/webcomponents/custom-elements/pull/88) on the custom-elements polyfill.

**Comment by ShaggyDude** [comment]
What's the status on this?

**Comment by seaneking** [comment]
Porting all SimpleElements to wc v1 is still blocked by that PR. I'm also flat out at the moment, but would happily accept PR for making simple-modal itself a hybrid Polymer 1/2 element in the meantime. Should be pretty straightforward.
